src/components/FeedbackButton/FeedbackButton.scss

```diff
@@ -1,32 +1,50 @@
 @import '~style/variables';
 
-.buttonWrapper {
-  position: fixed;
-  right: 0;
-  top: 50%;
-  margin-top: -30px;
-  margin-right: -50px;
-  z-index: $z-index-feedback-button;
-}
-
 .button {
```

Comment:
This isn't aligned in the centre. But if Nick's happy I'm happy.
![image](https://user-images.githubusercontent.com/4443482/33575644-6f7e3a1a-d991-11e7-8d36-e4999aabb6f5.png)